src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -932,6 +932,121 @@ public static void GetFullPath_Windows_Wildcards(char wildcard)
             AssertExtensions.Throws<ArgumentException>("path", null, () => Path.GetFullPath("test" + wildcard + "ing"));
         }
 
+        [Fact]
+        public static void GetFullPath_BasePath_InvalidInput()
+        {
+
+            Assert.Throws<ArgumentNullException>(() => Path.GetFullPath("", null));
+            Assert.Throws<ArgumentNullException>(() => Path.GetFullPath("tmp", null));
+            Assert.Throws<ArgumentException>(() => Path.GetFullPath("", "foo\bar"));
+            Assert.Throws<ArgumentException>(() => Path.GetFullPath("tmp", "foo\bar"));
+        }
+
+        public static IEnumerable<object[]> GetFullPath_BasePath_BasicExpansions_TestData()
+        {
+            string curDir = Directory.GetCurrentDirectory();
```

Comment:
Using the current directory is less deterministic than using a constant here, so the tests can possibly break in some weird case.